src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 summary: Wizard splits complex tasks and process into steps.
 ---
 
-# Wizard
+# Wizard UI Pattern
+You can use the Wizard UI pattern to split large complex tasks and processes into smaller manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
 
-Split complex tasks and process into steps.
+**How to Use the Wizard UI Pattern**
 
-Use the Wizard to present numbered steps or conditions that the end user needs to complete or reach a goal. Use it to split large tasks into more manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
+1. In Service Studio, in the Toolbox, search for  `Wizard`. 
 
-**How to use**
+    The Wizard widget is displayed.
+    
+     ![](<images/wizard-image-23.png>)
+
+1. From the Toolbox, drag the Wizard widget onto your application's screen.
 
-1. Drag Wizard pattern into the preview.
+    By default, the Wizard widget contains three Wizard Item blocks. Each Wizard Item represents a step. You can add or delete Wizard Items as required.
 
     ![](<images/wizard-image-1.png>)
 
-1. Drag as many WizardItems in the Content placeholder as required. 
+**Create a Wizard with Navigation Buttons**
 
-1. Set the content in the placeholders.
+The following example demonstrates how you can create a three step Wizard with navigation buttons.
 
-1. Set the mandatory values.
+1. Rename each of the Wizard Items to the following: 
+    * Shopping Details
+    * Payment Details
+    * Review Order
 
-    ![](<images/wizard-image-2.png>)
+        ![](images/wizard-image-14.png)
 
-1. Publish and test.
+1. Create an Input Parameter with the following properties:
```

Comment:
Add an Input Parameter to your screen. The Input Parameter controls the current step of the Wizard - meaning, the Wizard step that will be shown to the user. 
Setting the **Default Value** to 1 ensures the Wizard always starts at step 1.
We will change it when we are ready to show the next steps.

Reply:
Added a step about adding the Input Parameter. Edited the step re the properties. 

I'm not sure what 'We will change it when we are ready to show the next steps.' means

Reply:
Foreshadowing - why are we creating this InputParameter in the first place?

Reply:
It is explained here: 
In this example we call the input parameter **CurrentStep**. The CurrentStep input parameter controls the Wizard step that is shown to the user. Setting the Default Value to 1 ensures the Wizard always starts by showing step 1.